### ISSUE TITLE
Use foreign_field / foreign_table_field for configuration

### ISSUE THREAD

**Issue by nhovratov** [lochmueller/calendarize#804]
The inline field `calendarize` does not make use of modern relation fields foreign_field and foreign_table_field. Relations are stored as CSV list. This is probably due to the age of this extension. I would highly recommend to migrate this field (Needs UpgradeWizard). By doing so we benefit from proper reference index entries and generally better support for state of the art relations.

https://github.com/lochmueller/calendarize/blob/160ff2d952c7f2e5c385b38440cb91008c205fb4/Classes/Register.php#L102-L114
https://docs.typo3.org/m/typo3/reference-tca/main/en-us/ColumnsConfig/Type/Inline/Properties/ForeignField.html
https://docs.typo3.org/m/typo3/reference-tca/main/en-us/ColumnsConfig/Type/Inline/Properties/ForeignTableField.html

**Comment by lochmueller** [comment]
Yes, you are right. Feel free to write an upgrade wizard (also for tables, that use calednarize in your own models).
Regards,
Tim

**Comment by nhovratov** [comment]
I already tried to override it, but the Indexer still expects a CSV value and writes incorrect index records. So this place also needs to be adjusted. Maybe I can have a look someday, but right now I am very busy with Content Blocks. Btw it works really great together to create a custom event type on top of the default event. Maybe I can write I guide about that.

Thanks for the great extension! Really like the possibility to extend with own Models :+1:
